examples/overlay-warnings/README.md

```diff
@@ -0,0 +1,11 @@
+# Overlay
+
+```shell
+node ../../bin/webpack-dev-server.js --open
+```
+
+## What should happen
+
+The script should open the browser and show a heading with "Example: overlay with warnings".
+
+In `app.js`, make a syntax error. The page should now refresh and show a full screen error overlay that shows the syntax error.
```

Comment:
Got it. You have to do

```javascript
require("./casesensitive.js");
require("./caseSensitive.js");
```

It requires multiple, different imports pointing to the same place in order to work. This is enough for me. 👍